lm_eval/tasks/__init__.py

```diff
@@ -234,54 +316,26 @@ def get_task_name_from_object(task_object):
         else type(task_object).__name__
     )
 
-
 # TODO: pass num_fewshot and other cmdline overrides in a better way
-def get_task_dict(task_name_list: List[Union[str, Dict, Task]], **kwargs):
-    config = {**kwargs}
+def get_task_dict(task_name_list: List[Union[str, Dict, Task]], task_manager: TaskManager):
```

Comment:
Note this now requires a user to pass `task_manager`. This should be okay, if we document it enough